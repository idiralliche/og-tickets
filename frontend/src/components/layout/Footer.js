import React from 'react';

const Footer = () => {
  return (
    <footer className='footer'>
      <p>
        Simulation: Application vente de billets pour les Jeux Olympiques 2024
      </p>
    </footer>
  );
};

export default Footer;
