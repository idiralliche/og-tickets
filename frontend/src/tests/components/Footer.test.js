import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from '../../components/Footer';

describe('Footer component', () => {
    test('renders footer element', () => {
        render(<Footer />);
        const footer = screen.getByRole('contentinfo');
        expect(footer).toBeInTheDocument();
    });

    test('matches snapshot', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});
