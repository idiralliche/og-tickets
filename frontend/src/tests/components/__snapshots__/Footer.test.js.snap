// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p>
      © 2024 Jeux Olympiques - Tous droits réservés
    </p>
  </footer>
</DocumentFragment>
`;
